__tests__/app.test.js

```diff
@@ -41,19 +41,50 @@ describe("API Testing", () => {
 	});
 	describe("Articles Endpoint", () => {
 		test("Get: 200, should return a single article", () => {
-			return request(app).get("/api/articles/1").expect(200);
+			return request(app)
+				.get("/api/articles/3")
+				.expect(200)
+				.then(({ body }) => {
+					const { article } = body;
+					expect(article).toBeInstanceOf(Array);
+					expect(article.length).not.toBeLessThan(1);
+					article.forEach((article) => {
+						expect(article).toHaveProperty("author");
+						expect(article).toHaveProperty("title");
+						expect(article).toHaveProperty("article_id");
+						expect(article).toHaveProperty("body");
+						expect(article).toHaveProperty("topic");
+						expect(article).toHaveProperty("created_at");
+						expect(article).toHaveProperty("votes");
+						expect(article).toHaveProperty("article_img_url");
+					});
+				});
 		});
 	});
 });
-// Implement tests later when more methods have been implemented
+
 describe("Error Testing", () => {
 	describe("Endpoint errors", () => {
-		test("Get: 404, Returns error 404 when given an invalid url", () => {
-			return request(app).get("/api/topic").expect(404).then((response) => {
-				console.log(response.body);
-			})
+		test("Err: 404, Returns error 404 when given an invalid url", () => {
```

Comment:
This is not a 404 error - should /topic ever be an endpoint? (not to be confused with previous /topics)